src/vmaware.hpp

```diff
@@ -8929,11 +8317,53 @@ struct VM {
 #pragma warning(default: 4459)
 
         typedef NTSTATUS(__stdcall* PNtQuerySystemInformation)(ULONG, PVOID, ULONG, PULONG);
+
+#if defined(_MSC_VER)
+#define LIKELY(x)   (x)
+#define UNLIKELY(x) (x)
+#else
+#define LIKELY(x)   __builtin_expect(!!(x), 1)
+#define UNLIKELY(x) __builtin_expect(!!(x), 0)
+#endif
```

Comment:
we discussed this earlier but i didn't see the usage of it on line 8425, we can keep it